golf2/src/nvcounter_syscall.rs

```diff
@@ -55,7 +55,9 @@ impl<'c, C: NvCounter<'c>> NvCounterSyscall<'c, C> {
     /// value between 0 and the previous value.
     #[allow(unused)]
     pub fn initialize(&self) {
+        debug!("Initializing NvCounterSyscall.");
```

Comment:
Hmm, I think the name `initialize()` is misleading here. You're really initializing the counter, not the NvCounterSyscall interface to the counter.

Perhaps this message should be "NvCounterSyscall initializing counter".